omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -52,8 +68,10 @@ public void get(@RequestParam("encounter") Encounter encounter,
         model.addAttribute("showPatientHeader", showPatientHeader);
 
         HtmlForm htmlForm = htmlFormEntryService.getHtmlFormByForm(encounter.getForm());
+        
         if (htmlForm == null) {
-            throw new IllegalArgumentException("encounter.form is not an HTML Form: " + encounter.getForm());
+        	 log.warn("Active drugs are cannot be editted");
```

Comment:
shouldn't this be "encounter.form is not an HTML Form"?  that's the real error here, not that active drugs can't be edited.

Reply:
Resolved thanks